src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SuggestionModeCompletionProviderTests.cs

```diff
@@ -391,7 +391,7 @@ static void Main(string[] args)
         Console.CancelKeyPress += new ConsoleCancelEventHandler(((a$$
     }
 }";
-            await VerifyBuilderAsync(markup);
+            await VerifyNotBuilderAsync(markup);
```

Comment:
Yes, the first case is just above (called `IncompleteParenthesizedLambdaArgument`).